CHANGELOG.md

```diff
@@ -76,6 +76,7 @@ if input key is empty, or input data contains empty key.
   * via legacy `msg.Type()` (e.g. `message.action='send'`) is being deprecated, new `Msg`s won't emit these events.
   * Please use concrete `Msg` TypeURLs instead (e.g. `message.action='/cosmos.bank.v1beta1.MsgSend'`).
 * [\#9291](https://github.com/cosmos/cosmos-sdk/pull/9291) Migration scripts prior to v0.38 have been removed from the CLI `migrate` command. The oldest supported migration is v0.39->v0.42.
+* [\#9492](https://github.com/cosmos/cosmos-sdk/pull/9492) Bump `gov` and `upgrade` proto packages from `v1beta1` to `v1`. If you are using these proto files, please make sure to update the proto files locally.
```

Comment:
How about having a special section: _Proto Changes_ This will highlight more the changes for protobuf files.

Reply:
Yeah, a proto section makes sense 👍 . I also updated https://github.com/cosmos/cosmos-sdk/issues/9116 to remind ourselves to audit this section before the release